build.gradle

```diff
@@ -616,6 +648,10 @@ compileJava {
     ]
 }
 
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
```

Comment:
Unneeded it is like 10 lines above this.

Reply:
I did wonder about that, must be a merge error